.github/workflows/pr_ghaction.yml

```
@@ -0,0 +1,64 @@
+# Run all tests as GitHub Actions
+name: Unit Tests
```

```suggestion
name: PR Tests
```

This workflow includes spec + 💪acceptance💪 tests.